src/Microsoft.DotNet.Darc/src/Darc/Helpers/RemoteFactory.cs

```diff
@@ -30,7 +30,8 @@ public static IRemote GetRemote(CommandLineOptions options, string repoUrl, ILog
         {
             DarcSettings darcSettings = LocalSettings.GetDarcSettings(options, logger, repoUrl);
 
-            if (string.IsNullOrEmpty(darcSettings.GitRepoPersonalAccessToken))
+            if (darcSettings.GitType != GitRepoType.None &&
```

Comment:
>if [](start = 12, length = 2)

For own education purposes, when is GitType None? As in, in what scenarios?

Reply:
BAR only remotes.

Reply:
Of course

---
In reply to: [261309691](https://github.com/dotnet/arcade-services/pull/176#discussion_r261309691) [](ancestors = 261309691)